### ISSUE TITLE
Install linux on a common laptop and document it. 

### ISSUE THREAD

**Issue by ck2qsuZT** [cloyne/common-electronics#6]
Chromebooks are great but sometimes member need a more powerful word editor like libreoffice or other features chromeos doesn't offer. We currently have a chromebook but might get two more. What if we install Xubuntu on one of them?

**Comment by mitar** [comment]
I would advice against it. Let's just get some old PC for that. Or we could have that on the makerspace computer. Let's keep Chromebooks simple. You know how much work you made when you tried to install Linux on your laptop.

**Comment by ck2qsuZT** [comment]
Hey :) the issue wasn't installing linux on a chromebook, it was installing gentoo linux on a chromebook through LVM, dm-crypt with luks, and boot to ram. Also, brand new hardware that wasn't in the official Linux kernel at the time and still isn't fully supported. I got it figured out eventually:) I've installed Linux on five chromebooks so far and only had issues with the pixel. But yeah, it would be much easier to get a cheap laptop for about $100, install xubuntu on it, and keep the chromebooks as they are. But I'll wait for more than two complaints in two semesters, especially since we're getting the common desktop which will have these features.
